install_config/configuring_sdn.adoc

```diff
@@ -142,7 +143,39 @@ networkConfig:
 *redhat/openshift-ovs-multitenant* for the *ovs-multitenant* plug-in, or
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 <4> Service IP allocation for the cluster
-====
+
+Alternatively, you can create a pod network with multiple CIDR ranges by
+adding separate ranges into the `clusterNetworks` field with the range and the `hostSubnetLength`. 
+
+Multiple ranges can be used at once, and the range can be expanded. Pods can
```

Comment:
range can be expanded, and if no nodes are using a range, contracted.

"Pods can be moved" => "Nodes can be moved from one range to another by draining, deleting, and re-creating the node [see link for more details]".